lib/Mail.php

```diff
@@ -397,7 +397,7 @@ public function setParams(array $params)
     }
 
     /**
-     * Sets a single parameter for the email view and the Placeholders
+     * Sets a single parameter to the request object and the Placeholders
```

Comment:
Please adapt the warning message as the code says $key is allowed to be a string

Logger::warn('$key has to be a string - Param ignored!');